packages/react-data-grid/src/common/types.ts

```diff
@@ -10,9 +10,14 @@ export interface Column<T = unknown> {
   //FIXME: width and left should be optional
   width: number;
   left: number;
+  hidden?: boolean;
   cellClass?: string;
+  events?: {
+    [key: string]: (e: Event, info: ColumnEventInfo) => void;
```

Comment:
Should type it as `undefined | ((e: Event, info: ColumnEventInfo) => void);` to be safe